middleware.js

```diff
@@ -0,0 +1,55 @@
+const debug = require('debug')('hydra-box:middleware')
+const absoluteUrl = require('absolute-url')
+const { Router } = require('express')
+const rdf = { ...require('@rdfjs/data-model'), ...require('@rdfjs/dataset') }
+const rdfHandler = require('@rdfjs/express-handler')
+const setLink = require('set-link')
+const apiHeader = require('./lib/middleware/apiHeader')
+const iriTemplate = require('./lib/middleware/iriTemplate')
+const operation = require('./lib/middleware/operation')
+const resource = require('./lib/middleware/resource')
+
+function middleware (api, store) {
```

Comment:
So the `store` is a combination of RDF/JS [`Store`](http://rdf.js.org/stream-spec/#store-interface) and [`Source`](http://rdf.js.org/stream-spec/#source-interface)